doc/parallel-spec-execution.md

```diff
@@ -43,3 +43,6 @@ Link the program with `-threaded` and pass `+RTS -N -RTS` when running it:
 
 All specs that are not explicitly marked with `parallel` are run in the
 application's main thread.
+
+
+By default, the number of threads available for parallel execution is equal to the number of processors available to the process, as determined by `+RTS -N<n>`.  A different number (higher or lower) can be specified using the `-j` flag.  Note that this number is in addition to the main thread, and must be at least 1 (otherwise, parallelizable jobs would not be able to run).
```

Comment:
I think I would remove the note about the about `-j` having to be `>= 1`. If we have a proper warning we don't need a note here. (Maybe we should add a test for having a good warning.)

And this file seems to be word-wrapped.